docs/advanced-features/measuring-performance.md

```diff
@@ -181,6 +181,38 @@ export function reportWebVitals(metric) {
 >
 > Read more about [sending results to Google Analytics](https://github.com/GoogleChrome/web-vitals#send-the-results-to-google-analytics).
 
+## Web Vitals Attribution
+
+When debugging issues related to Web Vitals, it is often helpful to obtain more in-depth information from real users in the field.
```

Comment:
I think we need to state earlier what we're "attributing" - it sounds like we're tracking user data

Reply:
Fixed wording to explain "attribution" in detail and removed ambiguous description of sending RUM data.